include/alp/reference/blas3.hpp

```diff
@@ -1772,13 +1630,8 @@ namespace alp {
 			return SUCCESS;
 		}
 
-		RC rc = internal::set_band< 0 >( C, val );
-
-		if( rc == SUCCESS ) {
-			internal::setInitialized( C, true );
-		}
-
-		return rc;
+		internal::setInitialized( C, true );
```

Comment:
Shouldn't this depend on the init status of the A?

Reply:
Same answer as above.